Newsletters: add 252 (2023-05-24)

harding opened the pull request:
- [x] Lede, releases/RCs, topic links @harding 
- [x] Bitcoin Core 27021 @harding 
- [x] Waiting for confirmation 2 @glozow 
- [x] Client/Services @bitschmidty

LarryRuane commented on _posts/en/newsletters/2023-05-24-newsletter.md:
```diff
@@ -0,0 +1,175 @@
+    node almost immediately after downloading a very small amount of
+    data.
+
+    Regarding the usability of their prototytes, they write that "we
```
```suggestion
    Regarding the usability of their prototypes, they write that "we
```

LarryRuane commented on _posts/en/newsletters/2023-05-24-newsletter.md:
```diff
@@ -0,0 +1,175 @@
+    transfer of those specific tokens back to when they were created.
+    In an ideal scenario, that history grows linearly with the number of
+    transfers.  But if Bob wants to pay Carol an amount higher than he
+    received from Alice, he will need to combine the amount of tokens he
+    received from Alice with an amount of tokens he received in a
```
(IIUC)
```suggestion
    received from Alice, he will need to combine some tokens he
    received from Alice with some tokens he received in a
```

LarryRuane commented on _posts/en/newsletters/2023-05-24-newsletter.md:
```diff
@@ -0,0 +1,175 @@
+    validation, that's not strictly required but eventually becomes
+    effectively the case if merges are common.
+
+    That means a protocol which can compress the state of Bitcoin can
```
```suggestion
    That means a protocol that can compress the state of Bitcoin can
```

LarryRuane commented on _posts/en/newsletters/2023-05-24-newsletter.md:
```diff
@@ -0,0 +1,175 @@
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Bitcoin Core #27021][] Implement Mini version of BlockAssembler to calculate mining scores FIXME:harding (hoping glozow or Xekyo will volunteer, though)
```
If neither of them is available, I could help out. (I don't know this PR as well as they do, but I know it fairly well.)

Xekyo replied:
I can give it a spin.

Xekyo commented on _posts/en/newsletters/2023-05-24-newsletter.md:
```diff
@@ -0,0 +1,189 @@
+- [Core Lightning 23.05rc2][] is a release candidate for the next
+  version of this LN implementation.
+
+- [Bitcoin Core 23.2rc1][] is a release candidate for a maintenance
+  release of the previous major version of Bitcoin Core.
```
Is meanwhile released: https://github.com/bitcoin/bitcoin/releases/tag/v23.2

Xekyo commented on _posts/en/newsletters/2023-05-24-newsletter.md:
```diff
@@ -0,0 +1,189 @@
+- [Bitcoin Core 23.2rc1][] is a release candidate for a maintenance
+  release of the previous major version of Bitcoin Core.
+
+- [Bitcoin Core 24.1rc3][] is a release candidate for a maintenance
+  release of the current version of Bitcoin Core.
```
Meanwhile released: https://github.com/bitcoin/bitcoin/releases/tag/v24.1

Xekyo reviewed (commented):
LGTM so far

harding commented:
@Xekyo I push a substantive edit to your PR description.  Please feel free to revert or edit, of course.

Xekyo reviewed (commented):
Looks great, thanks for the polish!

Reviewed the complete text, no other comments.

glozow commented:
> @glozow pushed some edits to your section (rather than leaving comments) in the interest of saving time. Please feel free to revert. https://github.com/bitcoinops/bitcoinops.github.io/commit/f82617a8cf4ee7d91c220c63eef9c1c176ee3f1e

Makes perfect sense to me, thank you :)

harding commented:
Pushed edits for reviews (thanks!).  Added lede, updated releases/RCs, and add topic entries.  Reviewed @glozow @Xekyo @bitschmidty excellent additions, thanks!  Thanks everyone!

LarryRuane commented on _includes/specials/policy/en/02-cache-utility.md:
```diff
@@ -0,0 +1,82 @@
+
+As such, as part of their role as a cache of unconfirmed transactions,
+mempools also facilitate a public auction for inelastic block space:
+when functioning properly, the auction operates on free market
```
```suggestion
when functioning properly, the auction operates on free-market
```

LarryRuane commented on _includes/specials/policy/en/02-cache-utility.md:
```diff
@@ -0,0 +1,82 @@
+mirror those behaviors in order to maximize the utility of their
+mempools.
+
+The scarcity of blockspace exerts a downward pressure on the size of
```
```suggestion
The scarcity of blockspace exerts downward pressure on the size of
```

LarryRuane commented on _posts/en/newsletters/2023-05-24-newsletter.md:
```diff
@@ -0,0 +1,256 @@
+    That means a protocol that can compress the state of Bitcoin can
+    also be adapted to compress the state of a token's history, even one
+    where merges are common.  The authors describe how they would
+    accomplish that.  Their goal is to produce a proof that each
```
```suggestion
    accomplish that.  Their goal is to develop an algorithm that can produce a proof that each
```

LarryRuane commented on _posts/en/newsletters/2023-05-24-newsletter.md:
```diff
@@ -0,0 +1,256 @@
+  swap protocol resulting in lower fees and better privacy.
+
+- **Mutinynet announces new signet for testing:**
+  [Mutinynet][mutinynet blog] is a custom signet with 30 second block times that
```
```suggestion
  [Mutinynet][mutinynet blog] is a custom signet with 30-second block times that
```

bitschmidty reviewed (commented):
Took a couple of @LarryRuane's suggestions, fixed a topic entry, made a small change to the news section, and another to the "Waiting for confirmation" section
